vmServer/send/proto/Request.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+message TaskRequest {
+  string codePath=1;    //  pantheon path where the input code is stored
+  string dataPath=2;    //  pantheon path where the input files are stored
+  string outputPath=3;  //  pantheon path where the output will be stored
+  string targetPath=4;      //  path to file from code root that is to be executed
+  double timeout=5;     //  timeout for the execution 
+  repeated configPair actionPairs =6; //key value pairs containing action and related parameters
+}
+message configPair{ //key value pairs containing action and related parameters
```

Comment:
empty line before a message declaration, also for other declarations in the file

Reply:
Change has been made